json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9-]+$"
+    },
+    "frameworkStartDate": {
```

Comment:
If you're reasonably sure we won't want to be able to override these dates just for the purposes of output on the signature page (@TheDoubleK wasn't so sure) I'd agree.